azure/iac/KeyVault.rego

```diff
@@ -217,4 +217,356 @@ enablePurgeProtection_metadata := {
     "Resource Type": "microsoft.keyvault/vaults",
     "Policy Help URL": "",
     "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/vaults"
+}
+
+
+# PR-AZR-ARM-KV-006
```

Comment:
@shahinshirvani **any reason to skip the 004 and 005 and jump into 006?**
also title is duplicate with **PR-AZR-ARM-KV-010**

Reply:
@rezoan yes, the 004 and 005 are related to keyvault key and keyvault secret